pkg/storage/storage.go

```diff
@@ -1,48 +1,70 @@
 package storage
 
-import (
-	"fmt"
-)
+import "fmt"
 
 type Storage string
 
 const (
-	Bolt Storage = "bolt"
+	DBFilePrefix = "ssi-service"
+)
+
+var (
+	availableStorages = make(map[Storage]ServiceStorage)
```

Comment:
nit: I would prefer to initialize this in a `NewServiceStorage` method or similar

Reply:
Its purpose is automatic run initialize. i put this initialization to the `init()` before, but we cannot guarantee this `init()` was the first execution (another `init()` was in the `bolt.go` ). That's why, in the end, I've made a choice.
if we create a function like `NewServiceStorage `, we should consider that when and where we call it, things back to the beginning. so think this approach was not necessary.